ServerCore/ServerCore.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <UserSecretsId>04bdbf00-8cf2-4132-9652-595fd71fdc98</UserSecretsId>
```

Comment:
How will we handle this? Can everyone have the same guid or do we have to do something else?

Reply:
Everyone can have the same guid - once people have this change and follow the instructions I put out with it their local secret stores will be created with this guild.